ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -378,6 +411,12 @@ function check_upgrade_status() {
             echo -e "Eclipsed Time: $(( ($end_time - $start_time) / 60 ))m\n"
             return 0
         fi
+        if check_ota_case_enabled "OCP-73352"; then
```

Comment:
I guess this block should be moved out of while loop, the test is only needed to be executed one time?

Reply:
The upgrading is already finished or have not start outside the while loop.

Reply:
If so, we need finding a way to control it's only executed once to avoid duplicated test in the while loop.

Reply:
Updated like below:
```
if [ "${wait_upgrade}" == "$(( TIMEOUT - 10 ))" ] &&  check_ota_case_enabled "OCP-73352"; then
```